src/exercises/Congratulations.js

```diff
@@ -28,53 +31,72 @@ export default function Congratulations({
     );
   }
 
+  useEffect(() => {
+    api.getUserDetails((data) => {
+      setUsername(data.name);
```

Comment:
Don't we have this data in the cookies? 
If we take it from there we don't have to go all the way to the server, which is slower.

Reply:
I changed it to get the data from the cookies instead :)

Reply:
I've updated it again - now I get the data from LocalStorage instead. This works better with the extension.

Reply:
Oh! do you also have access to local storage not only cookies?

Reply:
I can't get LocalStorage from zeeguu.org from the extension. But I can use the zeeguu-react LocalStorage function "userInfo()" if we inside of the extension also have a "name" in the extensions LocalStorage. So by making these changes, everything works in the extension, because we also have a LocalStorage with a name in the extension. If that makes sense :)

Reply:
ah! so you save the info in your own LocalStorage. That sounds good.